### ISSUE TITLE
Phantor backport to RK Linux 6.1 made by hbiyik

### ISSUE THREAD

**Issue by neofeo** [rockchip-linux/kernel#328]
Hi, this user had backported panthor. This has to be here on the main kernel fork.
It's  essential for the success of this platform to have true replacement for panfork. 

https://github.com/hbiyik/linux-rockchip

**Comment by Joshua-Riek** [comment]
I completely agree and hope to see Rockchip officially support a Panthor backport to the SDK.

**Comment by ccrutchf** [comment]
Did this end up going anywhere?  I am participating in a different embedded project and would be interested in a Panthor driver for 6.1.
